docs/docs.trychroma.com/pages/reference/cli.md

```diff
@@ -1,3 +1,17 @@
 ---
 title: CLI
 ---
+
+## Vacuuming
+
+Vacuuming shrinks and optimizes your database. In general, you only need to vacuum once, after upgrading from an older version of Chroma (<0.6). **It does not need to be run regularly**. Vacuuming blocks all reads and writes to your database while it's running, so it should only be run when absolutely necessary. In most cases, vacuuming will save very little disk space. We recommend shutting down your Chroma server before vacuuming (although it's not strictly required).
```

Comment:
The language here is a bit confusing; we shouldn't tell people that it will 'save very little disk space' - instead, let's swap it around to say "if you had an old collection .... " etc that it  will save a bunch of space, but after that it won't any more. i.e. tell people when they should use this rather than when they shouldn't.